libs/wingsdk/src/target-sim/api.inflight.ts

```diff
@@ -179,7 +179,7 @@ function isApiResponse(response: unknown): response is ApiResponse {
 function transformRequest(req: express.Request): ApiRequest {
   return {
     headers: sanitizeParamLikeObject(req.headers),
-    body: req.body,
+    body: Object.keys(req.body).length > 0 ? req.body : "",
```

Comment:
Are we sure `req.body` is an object and not e.g. a string?

Reply:
if the body is empty it'll be an empty object, by default, if the body is a non-empty string or a non-empty object `Object.keys(req.body).length` results in a number rather than 0 
(if the body is a string it counts the number of chars, so if it's empty by purpose it'll still return an empty string)